tools/translate_japanese_posts.sh

```
@@ -0,0 +1,309 @@
+5. Preserve all existing content and translations
+
+Note: This script requires the 'translator' tool to be installed.
+Install with: pip install aicoder-translator
+
+EOF
+}
+
+# Function to get language code from language name
+get_language_code() {
+    local language="$1"
+    local lower_language=$(echo "$language" | tr '[:upper:]' '[:lower:]')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)